Flashcards.fgaspar17/FlashcardsLibrary/DatabaseConfig.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+
+
+namespace FlashcardsLibrary;
+
+public class DatabaseConfig
+{
+    public void InitializeDatabase(string databaseName)
+    {
+        try
+        {
+            using IDbConnection connection = new SqlConnection(GlobalConfig.SetupConnectionString);
+
+            connection.Open();
+            string? db = connection.QuerySingleOrDefault<string>(@$"SELECT name FROM sys.databases WHERE name = '{databaseName}'");
+
+            if (db == null)
+            {
+                connection.Execute($"CREATE DATABASE {databaseName};");
```

Comment:
```suggestion
                connection.Execute($"CREATE DATABASE [{databaseName}];");
```

using the square brackets ensures that if a user enters a special character as their database name that an SQL exception isn't thrown